computer-und-recht.csl

```diff
@@ -5,7 +5,7 @@
     <title-short>CR</title-short>
     <id>http://www.zotero.org/styles/computer-und-recht</id>
     <link href="http://www.zotero.org/styles/computer-und-recht" rel="self"/>
-    <link href="https://www.zotero.org/styles/neue-juristische-wochenschrift" rel="template"/>
+    <link href="http://www.zotero.org/styles/neue-juristische-wochenschrift" rel="template"/>
```

Comment:
Why don't we normalize all URLs to use https? Wouldn't that make for a better default?